Components/Mollie/MollieShipping.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace MollieShopware\Components\Mollie;
+
+use MollieShopware\Gateways\MollieGatewayInterface;
+use Shopware\Models\Dispatch\Dispatch;
+use Shopware\Models\Order\Order;
+
+class MollieShipping
+{
+
+    /**
+     * Shopware tracking code variable of order.
+     */
+    const TRACKING_CODE_VARIABLE = '{$sOrder.trackingcode}';
```

Comment:
I don't know if it's relevant in this case but there is also $offerPosition.trackingcode: https://docs.shopware.com/de/shopware-5-de/tutorials-und-faq/trackingnummern-und-sendungsverfolgung

Reply:
we only support the overall order tracking number for now :)